docs/google-analytics.md

```diff
@@ -0,0 +1,78 @@
+## how google analytics works
```

Comment:
All of the headers should begin with a capital letter, just like a sentence would
```suggestion
## How google analytics works
```